src/test/java/uk/gov/di/ipv/service/ConfigurationServiceTest.java

```diff
@@ -72,4 +92,37 @@ void shouldReturnSameCredentialIssuersWhenBase64EncodingHasNotChanged() {
 
         assertSame(credentialIssuers1, credentialIssuers2);
     }
+
+    @Test
+    void usesLocalSSMProviderWhenRunningLocally(WireMockRuntimeInfo wmRuntimeInfo)
+            throws JsonProcessingException, InterruptedException {
+        stubFor(post("/").willReturn(ok()));
+        environmentVariables.set("IS_LOCAL", "true");
+        environmentVariables.set("AWS_ACCESS_KEY_ID", "ASDFGHJKL");
+        environmentVariables.set("AWS_SECRET_ACCESS_KEY", "1234567890987654321");
+        systemProperties.set(
+                "software.amazon.awssdk.http.service.impl",
+                "software.amazon.awssdk.http.urlconnection.UrlConnectionSdkHttpService");
+
+        Thread.sleep(5000);
+        Exception e =
+                assertThrows(
+                        Exception.class,
+                        () -> new ConfigurationService().getSsmProvider().get("any-old-thing"));
+
+        System.out.println(e.getClass());
+        System.out.println(e.getMessage());
+        System.out.println(e.getStackTrace());
+        System.out.println(getAllServeEvents());
+        System.out.println(getAllServeEvents().size());
```

Comment:
Do we need to keep this println() calls?

Reply:
Not at all. This is just be working out why the test wasn't passing when running with GitHub actions. Will be removing them.

Reply:
OK, should be good to go now